parts/kubernetesmasterresources.t

```diff
@@ -24,10 +24,25 @@
       "dependsOn": [
         "[concat('Microsoft.Network/publicIPAddresses/', variables('masterPublicIPAddressName'))]"
       ],
+      "kind": "Storage",
       "location": "[variables('location')]",
       "name": "[variables('masterStorageAccountName')]",
       "properties": {
-        "accountType": "[variables('vmSizesMap')[variables('masterVMSize')].storageAccountType]"
+        "encryption": {
+          "keySource": "Microsoft.Storage",
+          "services": {
+            "blob": {
+              "enabled": "true"
```

Comment:
As documented in the spec, "enabled" is a boolean type, do we need to use "true", or it can be just true.
It maybe ok, because SRP is mostly written in c# code. When unmarshal, string or boolean can both be handled nicely. 

However, it is not true for other unmarshal packages. Like in golang, it need to be exactly matched. So I suggest to use true, not "true"